interview-questions.md

```diff
@@ -7,57 +7,81 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
-
-  Researched answer:
-
+    Your answer: I understand that Rails follows an MVC pattern however, I can't quite recall the meaning of the acronym and I'd love to give you an accurate answer. If it's fine with you, I can look it up in my notes.
```

Comment:
Excellent answer. Even though you weren't able to provide a full answer, you asked to do some research through your own notes. 💯